grpc-example/grpc-example-scala/build.sbt

```diff
@@ -1,4 +1,8 @@
-import akka.grpc.gen.scaladsl.play.{ PlayScalaClientCodeGenerator, PlayScalaServerCodeGenerator }
+
+
+import play.grpc.gen.scaladsl.{ PlayScalaClientCodeGenerator, PlayScalaServerCodeGenerator }
+
+import play.grpc.gen.javadsl.PlayJavaServerCodeGenerator
```

Comment:
this seems unused and, considering it's `xxx.javadsl.xxx`, I'd say it's missplaced?